opentelemetry-appender-tracing/examples/basic.rs

```diff
@@ -15,9 +18,19 @@ fn main() {
         )]))
         .with_simple_exporter(exporter)
         .build();
-    let layer = layer::OpenTelemetryTracingBridge::new(&provider);
-    tracing_subscriber::registry().with(layer).init();
-
-    error!(name: "my-event-name", target: "my-system", event_id = 20, user_name = "otel", user_email = "otel@opentelemetry.io", message = "This is an example message");
+    let env_filter = EnvFilter::from_default_env();
```

Comment:
this example is to show the usage of appender-tracing, and we can leave it untouched. We need to fix https://github.com/open-telemetry/opentelemetry-rust/tree/main/examples/self-diagnostics to showcase this, if needed.
I am unsure if we need to show this with a full blown example, but no harm doing it in https://github.com/open-telemetry/opentelemetry-rust/tree/main/examples/self-diagnostics